packages/core/src/typegenTypes.ts

```diff
@@ -167,7 +167,7 @@ type MergeWithInternalEvents<TIndexedEvents, TInternalEvents> = TIndexedEvents &
 
 type AllowAllEvents = {
   eventsCausingActions: Record<string, string>;
-  eventsCausingActors: Record<string, string>;
+  eventsCausingBehaviors: Record<string, string>;
```

Comment:
I'm glad that the renamed property stays at the same position when we consider the alphabetical order 🤣